src/main/java/com/beerjournal/breweriana/item/ItemService.java

```diff
@@ -34,4 +34,21 @@ ItemDto addItem(String ownerId, ItemDto itemDto) {
         return itemDto;
     }
 
+    ItemDto deleteItem(String ownerId, String itemId) {
```

Comment:
Well to honest I only know this approach :P. You could probably implement it a little bit better (e.g. with custom annotation). As for security config I think you can only exclude certain endpoint for groups or authorized users but I don't think there is a possibility to check for example if you own a collection as it requires some business logic :thinking: 

I may be wrong on this, but either way I think we should check with SecurityUtils for now and replace it in the future if we find a better way.